actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -931,7 +931,9 @@ def deep_dup
     #   params = ActionController::Parameters.new(id: "1_123", tags: "ruby,rails")
     #   params.extract_value(:id) # => ["1", "123"]
     #   params.extract_value(:tags, delimiter: ",") # => ["ruby", "rails"]
+    #   params.extract_value(:non_existent_key) # => []
     def extract_value(key, delimiter: "_")
+      return [] if @parameters[key].nil?
       @parameters[key].split(delimiter)
```

Comment:
```suggestion
      @parameters.fetch(key) { return [] }.split(delimiter)
```

Might as well avoid hashing (and casting) the key twice.

Reply:
```suggestion
      @parameters.fetch(key, "").split(delimiter)
```
Although the split method is called redundantly, this is simpler. What do you think?
This method also avoids hashing the key twice.